### PR TITLE
fix(youtube/spoof-signature-verification): additional fixes for subtitle locations

### PR DESCRIPTION
**Pull request by LisoUseInAIKyrios**
I saw there is a third set of incorrect subtitle parameters.

The subtitle fix could always override non-shorts with a single default value (34, 50, 95), but that would override any custom subtitle positions a video may have.

For now, this PR seems to fix all situations.

### CODE REVIEW

File: app/src/main/java/app/revanced/integrations/patches/SpoofSignatureVerificationPatch.java [hunk 1]
```diff
@@ -76,62 +76,91 @@ public static void onResponse(int responseCode) {
...
+     * Known incorrect default Shorts subtitle parameters, and the corresponding (non-Shorts) correct values.
+     */
+    private static final SubtitleWindowReplacementSettings[] knownReplacements = {
+            new SubtitleWindowReplacementSettings(9, 20, 0, true, false,
+                    34, 50, 90),
+            new SubtitleWindowReplacementSettings(10, 50, 0, true, false,
+                    34, 50, 95),
+            new SubtitleWindowReplacementSettings(9, 20, 0, true, true,
+                    33, 20, 100),
+    };
```
Comment (oSumAtrIX) [inline app/src/main/java/app/revanced/integrations/patches/SpoofSignatureVerificationPatch.java:92]: At this point probably an enum is a better choice.

**Review by oSumAtrIX — APPROVED** [review]

File: app/src/main/java/app/revanced/integrations/patches/SpoofSignatureVerificationPatch.java (hunk 1 above)
Comment (oSumAtrIX) [reply app/src/main/java/app/revanced/integrations/patches/SpoofSignatureVerificationPatch.java:92]: Can a name be inferred from the values? It is known what those values do, maybe a name can be inferred from that for each combination of values.

Comment (oSumAtrIX) [reply app/src/main/java/app/revanced/integrations/patches/SpoofSignatureVerificationPatch.java:92]: Some name along these lines sounds fine to me.

**Comment by SodaWithoutSparkles** [comment]
> [...] but that would override any custom subtitle positions a video may have.

Some videos might need this to differentiate between multiple speakers speaking at the same time, or not to block some important information that just happens to be at the bottom. Overriding any custom value might not be a good idea.

**Pull request merged**